app/src/Homepage/Homepage-components.tsx

```diff
@@ -22,10 +43,19 @@ export const Sidebar: React.FC<SidebarProps> = ( props: SidebarProps ) => {
 
 
 export const LevelTab: React.FC<{ classlevel: string }> = ({ classlevel }) => {   
+    const navigate = useNavigate();
     
     const handleClick = () => {
-        console.log('Button clicked: ' + classlevel);
-        
+        if (classlevel === 'Home') {
+            // If 'Home' button, route to default homepage
```

Comment:
Nit: Just say "Default route"

Look at below comment to handle nits in PRs

Reply:
changed